.travis.yml

```diff
@@ -46,6 +46,19 @@ matrix:
         - DESC="findbugs and pmd"
         - CMD="mvn clean compile pmd:check findbugs:check"
 
+    # checkstyle injection compile no error (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="checkstyle injection compile no error"
+        - CMD1="git clone https://github.com/checkstyle/checkstyle &&"
+        - CMD2="cp src/main/resources/com/github/checkstyle/regression/extract/ExtractInfoGeneratorTest.java "
+        - CMD3="checkstyle/src/test/java/com/puppycrawl/tools/checkstyle/ExtractInfoGeneratorTest.java &&"
+        - CMD4="cp src/main/resources/com/github/checkstyle/regression/extract/Json.java "
+        - CMD5="checkstyle/src/test/java/com/puppycrawl/tools/checkstyle/Json.java &&"
+        - CMD6="cd checkstyle &&"
+        - CMD7="mvn clean compile"
+        - CMD="$CMD1$CMD2$CMD3$CMD4$CMD5$CMD6$CMD7"
```

Comment:
@Luolc no exception test is fine for now.
Travis won't be verifying output for now. We can manually confirm this.

Later since this extract program should work on all checkstyle versions, we could use release for code coverage and expected output in another issue.